graphql_compiler/tests/test_helpers.py

```diff
@@ -281,14 +284,35 @@ def get_schema():
     return schema
 
 
+def get_schema_graph():
```

Comment:
Is this a specific OrientDB-based schema graph, or should it be interchangeable with the rest?